web/libs/editor/src/tags/object/Paragraphs/HtxParagraphs.jsx

```diff
@@ -487,6 +487,8 @@ class HtxParagraphsView extends Component {
     requestAnimationFrame(() => {
       const container = this.myRef.current;
       const mainContentView = document.querySelector(this.mainContentSelector);
+
+      if (!mainContentView) return;
```

Comment:
this doesn't seem to be Button related, does this address an existing issue? how does `mainContentView` get into a state where it's falsey?